flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/connectors/hive/HiveSource.java

```diff
@@ -60,7 +62,7 @@
  */
 @PublicEvolving
 public class HiveSource<T> extends AbstractFileSource<T, HiveSourceSplit>
-        implements DynamicParallelismInference {
```

Comment:
shouldn't the hive connector changes do into [https://github.com/apache/flink-connector-hive](https://github.com/apache/flink-connector-hive) . I assume the hive connector has moved out of Flink core and the hive connector folder in core is not being maintained.